lib/view_model/after_auth_view_models/settings_view_models/app_setting_view_model.dart

```diff
@@ -14,10 +14,11 @@ class AppSettingViewModel extends BaseModel {
   ///   None
   ///
   /// **returns**:
-  ///   None
-  Future<void> logout() async {
+  /// * `Future<bool>`: A [Future] that resolves to a [bool] value indicating whether the user has been logged out.
```

Comment:
Write a small non-redundant sentence.
Like: "Logs the user out and returns logout status"

Reply:
@literalEval updated.